arbitrum-docs/build-decentralized-apps/arbitrum-vs-ethereum/02-block-numbers-and-time.md

```diff
@@ -17,7 +17,15 @@ As in Ethereum, Arbitrum clients submit transactions, and the system executes th
 
 Once in the chain's core inbox contract, transactions are processed in order. Generally, some time will elapse between the time when a message is put into the inbox (and timestamped) and the time when the contract processes the message and carries out the transaction requested by the message.
 
-In this page we describe what does this mechanism mean for the block numbers and the time assumptions of the transactions submitted to Arbitrum.
+Additionally, since the calldata of Arbitrum transactions (or the DAC certificate on AnyTrust chains) is posted to Ethereum, the gas paid when executing them includes an L1 component to cover the costs of the batch poster.
```

Comment:
```suggestion
Additionally, since the calldata of Arbitrum  transactions (or the DAC certificate on  <a data-quicklook-from="arbitrum-anytrust-chain">AnyTrust</a>chains) is posted to Ethereum, the gas paid when executing them includes an L1 component to cover the costs of the batch poster.

```